contracts/TokenHub.sol

```diff
@@ -178,14 +116,6 @@ contract TokenHub is ITokenHub, System, IParamSubscriber, IApplication, ISystemR
         return actualAmount;
     }
 
-    function claimMigrationFund(uint256 amount) external onlyStakeHub returns (bool) {
```

Comment:
please dont just remove it, but revert

Reply:
can you check that `claimRewards` seems no longer used.

Reply:
> can you check that `claimRewards` seems no longer used.

yes, modify to "deprecated"

Reply:
> please dont just remove it, but revert

recovered